symplyphysics/laws/waves/photoelectric_effect.py

```diff
@@ -0,0 +1,37 @@
+from sympy import (Eq, solve)
```

Comment:
We have a convention of not using marketing names, like "Newton's first law" or such. Write law name as "what_we_get_from_what_we_put" or "value_is_something" for definitions.

Reply:
By the way, maybe this law should be in a `quantum` branch of laws?

Reply:
Changed filename to `photoelectron_energy_from_frequency.py`.

It seems to me, there're several files that should be in `quantum` folder, such as `photon_energy...` or `photon_momentum...`. Or may be it shoud be named `atomic`, for both particles and waves?